### ISSUE TITLE
WorldEdit not working after proper install

### ISSUE THREAD

**Issue by Oufufoyro** [SIsilicon/WorldEdit-BE#134]
> **Moderator Note:**
> This report covers users who are having problems using WorldEdit the first they are installing the addon on their game. If you're that user, please try the following first before commenting.
> 1. Check that you are using the latest addon update compatible with your Minecraft version
> 2. Restarting the game
> 3. Redownloading the addon
> 4. Reinstalling Minecraft
> 
> If none of these work, enable Content Logs and send a copy of what gets printed when entering the troublesome world.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behaviour

I don’t know what has been happening but it’s not letting me use the mod any more.
I can’t get the kit or use the axe for marking the box.

### Expected behaviour

To be able to use it like I use to.

### Steps to reproduce

I activate the behavior, resource and the global resource packs.
I activate all the needed experimental modes.
I get in the game and type in `/tag @s add worldedit` and try to use the equipment but it doesn’t work.

### Content logs

It says I don’t have any

### Minecraft version

 V1.19.51

### Device and operating system

bedrock edition

**Comment by CrackedMatter** [comment]
Are you using the latest version (0.7.0)?

**Comment by manuscrit109** [comment]
Hello! I also have the same problem.

I downloaded minecraft server for Bedrock from the Minecraft website, and installed the addon (the version made for servers) on my server.

Both are the latest version. Minecraft also is.

I made sure to add the tag, but the commands ;wand (or any other commands) command does not work.
I also tried with an wooden axe, and got no luck.

I currently have other addons installed ( Better Foliage, Chisels, Connected glass, Modern Furniture, WHITA) Don't know if it could be the issue. 

Thanks in advance!

**Comment by SIsilicon** [comment]
Closing since it lacks activity and information.

**Comment by Inshomothal** [comment]
Hello @SIsilicon,

I am having the same problem right now and I have the latest version (0.7.0). I just updated my game to Minecraft version 1.19.60 and the issue began after I updated my game.

<h3>Content logs</h3>
<p>It says I don’t have any</p>


<h3>Minecraft version</h3>
V1.19.51

<h3>Device and operating system</h3>

- bedrock edition
- Windows 11

<h3>Additional info</h3>

![Screenshot 2023-02-10 161301](https://user-images.githubusercontent.com/58829269/218225369-c9e6c07c-49e6-4db6-8f98-f3b8848b14b8.png)
![Screenshot 2023-02-10 160852](https://user-images.githubusercontent.com/58829269/218225411-28e4e294-2c4b-4cd9-b823-60894d539c2b.png)
![Screenshot 2023-02-10 160843](https://user-images.githubusercontent.com/58829269/218225405-cdd969fb-471a-49d1-b5e1-b076aa0ebb05.png)
![Screenshot 2023-02-10 160834](https://user-images.githubusercontent.com/58829269/218225463-5cbe2af8-6529-4b42-9d9f-700edc006006.png)
![Screenshot 2023-02-10 160912](https://user-images.githubusercontent.com/58829269/218225494-5078ab7d-8142-440d-93ee-567bb6f55b01.png)
![Screenshot 2023-02-10 160923](https://user-images.githubusercontent.com/58829269/218225540-7f052c33-7210-4a53-b69e-ee85b5e46369.png)

**Comment by SIsilicon** [comment]
@Inshomothal the lates version is 0.7.1. 0.7.0 doesn't support Minecraft 1.19.60

**Comment by Inshomothal** [comment]
Thank you for the update

**Comment by pszalanski** [comment]
Same here. I tried the released version 0.8.0 as well as the current master, which I built locally.

Can I help with more info? Any logs I can post here?

**Comment by SIsilicon** [comment]
> Same here. I tried the released version 0.8.0 as well as the current master, which I built locally.
> 
> Can I help with more info? Any logs I can post here?

Yes, please post content logs when loading a world with it enabled.

**Comment by LilyTheCuteCatgirl** [comment]
welp same reason for closing as sisilicon on Jan 12, 2023 lacks activity
